app/javascript/mastodon/features/emoji/emoji_compressed.d.ts

```diff
@@ -15,7 +15,12 @@ export type Search = string;
  */
 export type Skins = null;
 
-export type FilenameData = string[] | string[][];
+export type Filename = string;
```

Comment:
Do we need to export all of those?

I dont see `Filename` imported anywhere for example

Reply:
Fixed. Thanks.